docs/AwsMarketplaceGuide.md

```diff
@@ -1,12 +1,14 @@
 # How to install Scalar products through AWS Marketplace
 
-Scalar products (ScalarDB, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
+Scalar products (ScalarDB Server, ScalarDB Cluster, ScalarDL, and their tools) are available in the AWS Marketplace as container images. This guide explains how to install Scalar products through the AWS Marketplace.
 
 Note that some Scalar products are available under commercial licenses, and the AWS Marketplace provides those products as pay-as-you-go pricing or a Bring Your Own License (BYOL) option. If you select pay-as-you-go pricing, AWS will charge you our product license fee based on your usage. If you select BYOL, please make sure you have the appropriate license for the product.
 
 ## Subscribe to Scalar products from AWS Marketplace
 
 1. Access to the AWS Marketplace.
+   * [ScalarDB Cluster Standard Edition (Pay-As-You-Go)](https://aws.amazon.com/marketplace/pp/prodview-jx6qxatkxuwm4)
+   * [ScalarDB Cluster Premium Edition (Pay-As-You-Go)](https://aws.amazon.com/marketplace/pp/prodview-djqw3zk6dwyk6)
    * [ScalarDL Ledger (Pay-As-You-Go)](https://aws.amazon.com/marketplace/pp/prodview-wttioaezp5j6e)
    * [ScalarDL Auditor (Pay-As-You-Go)](https://aws.amazon.com/marketplace/pp/prodview-ke3yiw4mhriuu)
    * [ScalarDB (BYOL)](https://aws.amazon.com/marketplace/pp/prodview-rzbuhxgvqf4d2)
```

Comment:
For ScalarDB (BYOL), it seems to be the correct name, but maybe we should call it `ScalarDB Server (BYOL)` to make it matchable with other docs?
Also, shouldn't it be marked as deprecated?
Or, is it going to be replaced with `ScalarDB Cluster (BYOL)`?

Reply:
@feeblefakie  
Thank you for your review!


> For ScalarDB (BYOL), it seems to be the correct name, but maybe we should call it ScalarDB Server (BYOL) to make it matchable with other docs?

On the AWS Marketplace page, the product name (page title) is `ScalarDB (BYOL)` at this time. So, I use the same name in the document.

Also, when we published this product, there was no other product like ScalarDB Cluster. There was ScalarDB Server only. So, I didn't state `ScalarDB Server` explicitly before.

I agree that we should explain `ScalarDB Server` explicitly now. But, first, we have to update the product name on AWS Marketplace.

So, I will do it first, and after that, I will update the product name from `ScalarDB (BYOL)` to `ScalarDB Server (BYOL)` in another PR.


> Also, shouldn't it be marked as deprecated?
> Or, is it going to be replaced with ScalarDB Cluster (BYOL)?

I think we should mark it as deprecated. (And, in the future, we can remove it from document.)
However, at this time, there is one minor problem.

Now, ScalarDB Cluster (Pay As You Go) is provided in AWS Marketplace only. In Azure Marketplace, ScalarDB Cluster is not published.

So, from the perspective of Azure Marketplace users, they cannot use ScalarDB Cluster. So, I think we should not mark ScalarDB Server as deprecated in Azure Marketplace Guide yet.

Also, I thought we can mark ScalarDB Server as deprecated in the AWS Marketplace Guide only. But, I also thought it causes some confusion since AWS Marketplace Guide states `it is deprecated` and Azure Marketplace Guide states `it is not deprecated`.

This is why I didn't mark ScalarDB Server as deprecated in the AWS Marketplace Guide.

However, I already explain `ScalarDB Server is deprecated` in several documents. So, now I'm thinking about it is OK to mark ScalarDB Server as deprecated in AWS Marketplace Guide. 

Also, I think we can add `[Deprecated]` mark in the AWS Marketplace page.

As the above, there is a bit complex concerns between the each Marketplace. I think we also must to consider how to provide ScalarDB Cluster for Azure users. So, could you please let me talk with you about these points later?

Reply:
@feeblefakie  
According to the discussion, I updated the product name on the AWS Marketplace side to `[Deprecated] ScalarDB Server (BYOL)`.
https://aws.amazon.com/marketplace/pp/prodview-rzbuhxgvqf4d2

Same as the above update, I updated this document in commit 3fb947ebd24be61fbe58b157e2abb19fdd422a1f.

And, we will keep this `[Deprecated] ScalarDB Server (BYOL)` for a while. (Remove it in the future after ScalarDB Server is completely deprecated.)

Also, after we implement BYOL license key feature, we will add a new product `ScalarDB Cluster (BYOL)` in the future.